android/java/org/chromium/chrome/browser/quick_search_engines/utils/QuickSearchEnginesViewsUtil.java

```diff
@@ -0,0 +1,72 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.quick_search_engines.utils;
+
+public class QuickSearchEnginesViewsUtil {
+
+    // private static View mQuickSearchEnginesView;
```

Comment:
Is it intended to be all commented?

Reply:
oops i missed it. I am removing it right away